lightning/src/onion_message/messenger.rs

```diff
@@ -569,7 +573,7 @@ where
 			},
 		};
 
-		log_trace!(self.logger, "Sending onion message {}", log_suffix);
+		log_trace!(self.logger, "Responding to onion message with path_id {:02x?} with: {:?}", reply_path, contents);
```

Comment:
Could you squash and rebase on `main`? A re-added fuzz test is failing since these messages were changed and it asserts on them. You'll need to updated accordingly.